app/controllers/notes_controller.rb

```diff
@@ -0,0 +1,55 @@
+class NotesController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @notes = current_user.notes
+  end
+
+  def new
+    @note = current_user.notes.new
+  end
+
+  def create
+    @note = current_user.notes.create(note_params)
+    respond_to do |format|
+      if @note.save
+        format.html { redirect_to [current_user, @note], notice: "Note has successfully created." }
```

Comment:
why you need `format` and why `format.html` in both cases?

Reply:
done